wazp/callbacks/dashboard.py

```diff
@@ -245,27 +253,49 @@ def create_buttons_and_message() -> html.Div:
         is_open=False,
         color="success",  # warning or danger
         style={
-            "margin-right": "10px",
+            "margin-right": "150px",
             "margin-left": "10px",
             "margin-top": "5px",
         },
     )
 
+    clipboard = dcc.Clipboard(
+        id="clipboard",
+        title="Copy full path to clipboard",  # tooltip text
```

Comment:
```suggestion
        title="Copy full path to exported data to clipboard",  # tooltip text
```
Maybe this is clearer?
Or maybe "Copy full path to exported data" (no "to clipboard")

Reply:
I now show the clipboard icon right next to the output path, so I left it as 'Copy full path', since I think it is clear enough and concise. But do let me know if you think otherwise!